Library/Formula/pjproject.rb

```diff
@@ -0,0 +1,20 @@
+class Pjproject < Formula
+  homepage "http://www.pjsip.org/"
+  url "http://www.pjsip.org/release/2.3/pjproject-2.3.tar.bz2"
+  sha256 "e7fa60a3b59424430145af90372282ca778449f7b68b77bb24a9cf75d94d5765"
+
+  depends_on 'openssl'
+
+  def install
+    system "./configure", "--prefix=#{prefix}"
+    system "make", "dep"
+    system "make"
+    system "make", "install"
+    mkdir bin
+    cp "pjsip-apps/bin/pjsua-x86_64-apple-darwin14.1.0", "#{bin}/pjsua"
```

Comment:
I addressed the last point, but I can't seem to get the architecture and OS X versions from Homebrew - `RbConfig::CONFIG` has `"target_cpu"=>"universal"` and `"host_os"=>"darwin14.0.0"`.